internal/operator/controller/contour/service.go

```diff
@@ -152,34 +208,65 @@ func DesiredContourService(contour *operatorv1alpha1.Contour) *corev1.Service {
 
 // DesiredEnvoyService generates the desired Envoy Service for the given contour.
 func DesiredEnvoyService(contour *operatorv1alpha1.Contour) *corev1.Service {
+	var ports []corev1.ServicePort
+	for _, port := range contour.Spec.NetworkPublishing.Envoy.ContainerPorts {
+		var p corev1.ServicePort
+		httpFound := false
+		httpsFound := false
```

Comment:
these don't look like they're ever being set

Reply:
These vars were removed in a different PR that recently merged.

Reply:
I think you just want to set `httpFound` and `httpsFound` to `true` within the `case` statements for each of them. I'm not quite sure what you mean about the other PR since they were added in this PR.

Reply:
@skriss I apologize, I got lost in the review so please disregard my comment above. I have updated the PR so the conditional loops through the ports, sets the correct bool, and applies the service port params.